pom.xml

```diff
@@ -187,6 +224,53 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+
+      <!-- CVE fix versions -->
```

Comment:
to keep the build without CVE will be difficult, but anyway at least once it is clean ;)